lib/core/src/Cardano/Wallet/Primitive/Types/Address.hs

```diff
@@ -88,8 +90,10 @@ import qualified Data.Text.Encoding as T
 --
 newtype Address = Address
     { unAddress :: ByteString
-    } deriving (Read, Show, Generic, Eq, Ord)
-      deriving anyclass (NFData, Hashable)
+    }
+    deriving (Generic, Eq, Ord)
+    deriving anyclass (NFData, Hashable)
+    deriving (Read, Show) via (Quiet Address)
```

Comment:
This makes the output of `show` much more concise for values of type `Address`.

Instead of:
```hs
Address { unAddress = "XXXX" }
```

We now have:
```hs
Address "XXXX"
```